src/app/components/ms-item/ms-item.component.html

```diff
@@ -0,0 +1,60 @@
+<div class="section" [class.has-nested]="nested" [class.has-nested2]="nested2">   
+    <div class="row no-gutters" *ngIf="data?.locus">
+        <span class="col-1" *ngIf="nested || nested2"></span>
+        <span class="col-1" *ngIf="nested2"></span>
+        <div class="col-1 col-label">
+            <span *ngIf="data?.locus" class="locus">
+                <evt-content-viewer [content]="data?.locus"></evt-content-viewer>
+                <span>:</span>
+            </span>
+        </div>
+        <div class="col-6 col-content">
```

Comment:
Why do you use "col-6" and "col-content"? Anyway, this information will be contained in a panel, so you should use all the available space.